Dockerfile

```diff
@@ -0,0 +1,16 @@
+
+FROM python:3.8-slim-buster as base
+WORKDIR /app
+COPY pyproject.toml .
+RUN pip install poetry
+RUN poetry install
+EXPOSE 5000
+
+FROM base as development
+ENTRYPOINT poetry run flask run -h 0.0.0.0 -p 5000
+
+FROM base as production
+ENV FLASK_ENV=production
```

Comment:
This is a sensible default, worth noting that this gets overridden if the `.env` file has a different value but that's on whoever passes that file in to ensure it's set for prod!